src/main/g8/modules/l0/src/main/scala/$package$/l0/Main.scala

```diff
@@ -16,7 +16,7 @@ import java.util.UUID
       ClusterId(UUID.fromString("517c3a05-9219-471b-a54c-21b7d72f4ae5")),
       version = BuildInfo.version
     ) {
-    def dataApplication: Option[BaseDataApplicationL0Service[IO]] = None
+    override def dataApplication: Option[Resource[IO, BaseDataApplicationL0Service[IO]]] = None
```

Comment:
are you overriding implementation, or implementing?

Reply:
What I mean by overriding is that there is an implementation for this method in the base trait/class and there is no requirement for implementing. Or that function doesn't have any default implementation so you have to implement it here (for this case I don't think we need to use the override modifier)

Reply:
If we do not provide this override flag, it throws the following:
```
16.76 [error] def dataApplication: Option[cats.effect.Resource[cats.effect.IO,org.tessellation.currency.dataApplication.BaseDataApplicationL0Service[cats.effect.IO]]] (defined in class CurrencyL0App)

16.76 [error]     def dataApplication: Option[BaseDataApplicationL0Service[IO]] = None

16.76 [error]         ^

16.76 [error] /code/codebase/dagchads-metagraph/modules/l0/src/main/scala/com/my/currency/l0/Main.scala:21:9: `override` modifier required to override concrete member:

```

Reply:
But we want also to show to users that we have the functions available, I know that we don't need to provide them, but we want to explicitly let them appear

Reply:
Ok it throws because it is the first case (you are actually overriding existing default implementation) so the modifier is necessary.